### DISCUSSION TITLE
Travel Mode Tracking

### DISCUSSION THREAD

**Discussion by Svagtlys** [Freika/dawarich#307] [Ideas]
Are there any plans to add tracking the mode of travel to Dawarich? Things like being able to see miles walked since a date, or have a breakdown on the miles in the stats page with the ability to exclude certain modes (e.g. flight) when looking at the total miles traveled per month graph?

**Comment by Freika** [comment]
I would like to implement it at some point, but there are multiple approaches possible, and one of them will require Dawarich to make an assumption of travel mode and user to confirm or fix it.

Also, currently, Dawarich operates with points only, meaning the routes you can see on the frontend, are only being rendered when the page is being loaded. For Dawarich to suggest a mean of travel it would need to know where the route started and ended so it could calculate speed, consider the road (or pedestrian path, or train tracks) to be able to make an assumption.

With that being said, it's an interesting topic and feature to have, but not an easy one, for now.

**Reply by dietrichmd** [reply]
Piggybacking on this, those of us with google takeout exports will have a field in the JSON that denotes the type of travel (plane, walking, driving, motorcycling, bicycling, etc) -- or at least mine does.  Can we not, at least in the interim, have that displayed?  Maybe in the future, a different color line based on travel method?  And then all the stuff @mikixd586 mentioned as well :)

**Comment by mikixd586** [comment]
I see from the reply that this is probably extremely hard to implement but it is for me one of the cooler uses for something like this. Google maps has a nice interface for this and it is nice how it shows you a breakdown of your traveling habits per month. Idk just kind of feels like THE feature for tracking service.
